File: loja/pedidos/src/main/java/br/com/bota/factory/AppMessage.java
package br.com.bota.factory;

import io.micronaut.context.MessageSource;
import io.micronaut.context.annotation.Bean;
import io.micronaut.context.annotation.Factory;
import io.micronaut.context.i18n.ResourceBundleMessageSource;

@Factory
public class AppMessage {
    @Bean
    public MessageSource messageSource() {
        return new ResourceBundleMessageSource("i18.messages_pt_BR");
    }
}

File: loja-kotlin/kclientes/src/main/kotlin/br/com/bota/factory/AppMessage.kt
package br.com.bota.factory

import io.micronaut.context.MessageSource
import io.micronaut.context.annotation.Bean
import io.micronaut.context.annotation.Factory
import io.micronaut.context.i18n.ResourceBundleMessageSource


@Factory
open class AppMessage {
    @Bean
    open fun messageSource() : MessageSource = ResourceBundleMessageSource("i18.messages_pt_BR")
}